microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -311,6 +313,18 @@ To provide feedback about the Ad Selection API origin trial, create a new issue
 * To submit a new issue, go to the [new issue form](https://github.com/WICG/privacy-preserving-ads/issues/new?labels=Ad+Selection%20API,OriginTrialFeedback&title=%5BAd%20Selection%20API%5D+Feedback).
 
 
+<!-- ====================================================================== -->
+## Terms of Use
+
+See [Terms of Use for Ad Selection API](/legal/microsoft-edge/ad-selection-api/terms-of-use).
```

Comment:
```suggestion
See [Terms of Use for Ad Selection API](/legal/microsoft-edge/ad-selection-api/terms-of-use).
```
add /en-us/ b/c Legal repo is by default English-only, not machine auto translated.  2x
@leahtu fyi